src/easylink/configuration.py

```diff
@@ -184,8 +184,12 @@ def _get_schema(self, potential_schemas: list[PipelineSchema]) -> PipelineSchema
         Returns
         -------
             The first ``PipelineSchema`` that validates the requested pipeline.
-            If no validated ``PipelineSchema`` is found, `exit()` is called with
-            `errno.EINVAL` and any validation errors are logged.
+
+        Raises
+        ------
+        SystemExit
+            If no requested pipeline is validated, the program exits with a non-zero
```

Comment:
I think line 177 above should say "... that validates the requested pipeline **configuration**."

And:

```suggestion
            If the pipeline configuration is not valid for any of the potential_schemas, the program exits with a non-zero
```